### PR TITLE
My last try at pulling: Defender and balancing

### PR DESCRIPTION
**Pull request by Danimal696**
Defender model complete with ogre vertex fails (totally needed) and
balancing of creatures, battles should go faster and be more deathly
now.

### CODE REVIEW

**Comment by akien-mga** [comment]
It worked ;-)

This pull request is fine and can be merged (once it's tested I mean, but on the form you're good).
The only thing that you might want to do differently next time is that you don't have to create a specific branch on your fork before doing the pull request: you can commit stuff directly in your fork in the development branch.

In this case you named your branch "https-/github.com/Danimal696/OpenDungeons", which doesn't make much sense; it should be either your main branch, development, or a specific branch if you prefer (e.g. "balancing").

**Comment by Danimal696** [comment]
if i use development it instantly merges it, i thougth i was only linking to my fork, but it turns out it creates a new branch, im using git for windows, not very intuitive.

**Comment by akien-mga** [comment]
Yes basically you were doing things right back then when you merged things instantly: the only issue is that your git clone was the main repo, so when you commited and push a changed, it went directly to OpenDungeons/OpenDungeons.

Now that your clone (the one you manage with git for windows) is your own fork (Danimal696/OpenDungeons), you can freely commit and push anything, it will always be on _your_ fork. That's how, afterwards, you can compare it to the main repo and ask for a freeze push.

So basically, as long as your clone is your own fork, you can do whatever you want, including pushing to development (because "development" will be the development branch in Danimal696/OpenDungeons, and not the one on the main repo).

I know it's a bit confusing at first, but once you're used to it, it's great :-)

**Comment by hwoarangmy** [comment]
Seems like you are getting good at using github ^^
Your PR works, the defender is there and he is pretty cool. But why didn't you separate the weapon from the creature ? It would allow to give him better weapons, even if only of one kind.

Concerning balancing, it seems better. Battles seem to be shorter, which is cool.
FYI, in my next PR, I will release a console command to force a creature level. That will help us on balancing once we have an arena test map ^^

EDIT: I don't merge because I let @Bertram25 or @oyvindln test a bit but for me, it is mergeable ^^

**Comment by Danimal696** [comment]
i thought spear users could be left like that (defender and lizarman) but most of the others will get upgradable weapons

**Comment by hwoarangmy** [comment]
ok, no problem. Anyway, we can change that later if we feel the need to.

**Comment by Bertram25** [comment]
Congrats for the first fully functional PR. :D 
I'll review the diff and test asap.

**Comment by Bertram25** [comment]
Seems you planned on not upgrade the defense when leveling. (Why not after all.)
Also the attack and HP values look saner, indeed.
If no big problems appear, I'd say we merge and why not after a few game release a 0.4.9 RC1!!! :D

**Comment by akien-mga** [comment]
> Trivial: Here there seem to be a tab/space indentation problem.
> If you're not sure how to fix it easily, one of us will do it. Just tell. :)

To fix this, while on your "https-/github.com/Danimal696/OpenDungeons" branch, you can do the requested changes in config/spawnconditions.cfg and commit them (for example with a "Fix indentation" commit log). Then push to your repo, and the commit should appear in this pull request (the PR is linked to the branch of your repo from which it was made).

**Comment by Bertram25** [comment]
@akien-mga Thanks for the continuous inline help. :) I should have done so.

**Comment by Danimal696** [comment]
i decided to just give some fixed defense values based on relative tier and visual armor, for now itll make balancing easier without inmortal BigKnigths around ;), they are still hard to crack mind you; but even T1 creatures can now put a dent on them. Unchanging defense worked better than expected for this. In fact maybe it should just go up by 0.1 for everyone ; so in the long run its just a total of 3 more points added at level 30; remember shields and weapons will also affect it.

**Comment by Bertram25** [comment]
> i decided to just give some fixed defense values based on relative tier and visual armor, for now itll make balancing easier without inmortal BigKnigths around ;), they are still hard to crack mind you; but even T1 creatures can now put a dent on them. 

Eh eh. I don't know why @hwoarangmy is absolutely sure I'm fearing Big Knights but it seems to become viral. XD

> Unchanging defense worked better than expected for this. In fact maybe it should just go up by 0.1 for everyone ; so in the long run its just a total of 3 more points added at level 30; remember shields and weapons will also affect it.

Looks a perfectly sane logic. :) I'd say let's try the 0.1 for everyone in a second row as we might not need that for the next hopefully imminent release.
Ah, and yeah, I saw you coming with equipment. ;)

**Comment by akien-mga** [comment]
Defender is coming!
defender

**Comment by akien-mga** [comment]
For me this can be merged :-)

**Comment by hwoarangmy** [comment]
> Eh eh. I don't know why @hwoarangmy is absolutely sure I'm fearing Big Knights but it seems to become viral. XD

Might it come from the fact you have lowered 2 or 3 times their stats since they humiliated you ?

> Unchanging defense worked better than expected for this. In fact maybe it should just go up by 0.1 for everyone ; so in the long run its just a total of 3 more points added at level 30; remember shields and weapons will also affect it.

Yep, that seems a good idea. Anyway, IMHO, we should give a first try to balance everything (and not increasing defense during this period makes sense). And, when creatures will be more or less in a stable state (which should be the case with this PR), try to tune more deeply after playing a few games.

**Comment by Bertram25** [comment]
> Might it come from the fact you have lowered 2 or 3 times their stats since they humiliated you ?

This won't stand in a trial. :P

**Comment by hwoarangmy** [comment]
> This won't stand in a trial. :P

^^

**Comment by Bertram25** [comment]
Working. I merge :)
